sklearn/impute.py

```diff
@@ -1057,13 +1057,15 @@ class MissingIndicator(BaseEstimator, TransformerMixin):
         `missing_values` will be indicated (True in the output array), the
         other values will be marked as False.
 
-    features : str, optional
+    features : {"missing-only", "all", "not-constant"}, optional
```

Comment:
If find `not-constant` terrible... Please help me find a better name :) !

Reply:
It's mostly for internal use, so don't worry! But "some-missing" might be better.